cli/cmd/deploy/deploy.go

```diff
@@ -413,6 +418,28 @@ func createProjectFlow(ctx context.Context, client *adminclient.Client, req *adm
 	return res, err
 }
 
+func repoInSyncFlow(projectPath, branch string) bool {
+	syncStatus, err := gitutil.GetSyncStatus(projectPath, branch)
+	if err != nil {
+		// ignore errors since check is best effort and can fail in multiple cases
+		return true
+	}
+
+	msg := ""
+	switch syncStatus {
+	case gitutil.NA:
+		return true
+	case gitutil.SYNCED:
+		return true
+	case gitutil.MODIFIED:
+		msg = "Some files have been locally modified. These changes will not be present in deployed project. Confirm deploy."
+	case gitutil.AHEAD:
+		msg = "Local commits are not pushed to remote yet. These changes will not be present in deployed project. Confirm deploy."
```

Comment:
Does it look weird to have such long prompts? Maybe it should print these as info (except for `Confirm deploy.`), then have the prompt on the next line just say `"Do you want to continue"`?

Reply:
Yeah even I am also not sure. Having all msg in the prompt is good for context. Splitting it in multiple lines also seems good. I will split as you are suggesting.